ui/src/api/items/item.js

```diff
@@ -69,6 +70,11 @@ class QueryBuilder {
     return this;
   }
 
+  notOnLoan() {
+    this.notOnLoanQuery.push(`NOT circulation.state:ITEM_ON_LOAN`);
```

Comment:
the `loanActiveStates` have more, and I really wanted to use this one, because I want to force the librarian to check the inconsistency of the data, why does he have in hand a book which in the system is on loan.
The rest of the states like `In_Transit` are different, and I guess they could be "Force checkout" without checking the inconsistency. 
Although I based it on my intuition and it might be debatable

Reply:
discussed by voice :)